packages/edit-site/src/components/preferences-modal/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	PreferencesModal,
+	PreferencesModalTabs,
+	PreferencesModalSection,
+} from '@wordpress/interface';
+import { useMemo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import EnableFeature from './enable-feature';
+
+export default function EditSitePreferencesModal( {
+	isModalActive,
+	toggleModal,
+} ) {
+	const sections = useMemo( () => [
+		{
+			name: 'general',
+			tabLabel: __( 'General' ),
+			content: (
+				<PreferencesModalSection
+					title={ __( 'Appearance' ) }
+					description={ __(
+						'Customize options related to the block editor interface and editing flow.'
+					) }
+				>
+					<EnableFeature
+						featureName="focusMode"
+						help={ __(
+							'Highlights the current block and fades other content.'
+						) }
+						label={ __( 'Spotlight mode' ) }
+					/>
+				</PreferencesModalSection>
+			),
+		},
+		{
+			name: 'blocks',
+			tabLabel: __( 'Blocks' ),
+			content: (
+				<PreferencesModalSection
+					title={ __( 'Block interactions' ) }
+					description={ __(
+						'Customize how you interact with blocks in the block library and editing canvas.'
+					) }
+				>
+					<EnableFeature
+						featureName="keepCaretInsideBlock"
+						help={ __(
+							'Aids screen readers by stopping text caret from leaving blocks.'
+						) }
+						label={ __( 'Contain text cursor inside block' ) }
+					/>
+				</PreferencesModalSection>
+			),
+		},
+	] );
+	if ( ! isModalActive ) {
+		return null;
+	}
+	return (
+		<PreferencesModal closeModal={ toggleModal }>
+			<PreferencesModalTabs sections={ sections } />
```

Comment:
It does still feel a bit empty with one setting per tab. I wonder if right now an option could be to not add the tabs until there are more settings later on, and instead render the two `PreferenceModalSection`s directly in the modal.

I don't know if the styles will work out of the box, but it might be possible to make some quick fixes there.

Could be good to get some design feedback on it though.

Reply:
I think tabs will fill up faster now that we've modal.

Reply:
Should be pretty easy to get the block manager in there as well now.

I think the component needs to be refactored out of `edit-post`, but the rest of the implementation shouldn't be too difficult.

Reply:
Yeah, I added the second tab mainly thinking about the block manager which takes up a huge chunk of space 😄 

Plus with #38317 there'll be at least one more item under Appearance soon.